packages/dcos-ui-update-service/build

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 
+set -e  # Fail the script if anything fails
+set -x  # Verbose output
+set -u  # Undefined variables
+
 mkdir -p "$PKG_PATH"/bin
-mv "/pkg/src/dcos-ui-update-service/dcos-ui-update-service-1976d7e1d41307b71841c555fd08573c63bc80b0" "$PKG_PATH"/bin/dcos-ui-update-service
+mv "/pkg/src/dcos-ui-update-service/dcos-ui-update-service-afde1262ebf60aeb3b8431ad00d478d63633ae70" "$PKG_PATH"/bin/dcos-ui-update-service
```

Comment:
I'm not so keen on the need to change the hash in two places on every update. This could work with `dcos-ui-update-service-*` maybe?

Reply:
👍 I’ll change that next time we bump the version. We also have a task to implement semantic versioning.